app/lib/geordi-logger.coffee

```diff
@@ -31,7 +40,10 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
     newEntry = Object.assign {}, logEntry, @keys
     if GeordiLogger.tokens.indexOf(newEntry.projectToken) > -1
       @instance().logEvent newEntry
-      console.warn 'No Geordi logger available for event ', JSON.stringify(logEntry) unless @instance().logEvent
+      .catch (err) =>
+        if !@suppressWarnings
+          console.warn "Warning: Error was encountered logging to Geordi server [#{@state?.env} - #{@instance().GEORDI_SERVER_URL[@state?.env]}]:",err
```

Comment:
Where does `@state` come from? This isn't a React component afaict. Forgot to change after extracting to here maybe?